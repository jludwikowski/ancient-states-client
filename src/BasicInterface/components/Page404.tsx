import React from "react";

export const Page404: React.FunctionComponent = () => (
    <p>404 page not found! ;-( </p>
)
