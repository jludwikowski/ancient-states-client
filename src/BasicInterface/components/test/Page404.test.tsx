import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import { Page404 } from '../Page404';

describe('Page404', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Page404 />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('Should match snapshot', () => {
        const component = shallow(<Page404/>);
        expect(component).toMatchSnapshot();
    });
});
