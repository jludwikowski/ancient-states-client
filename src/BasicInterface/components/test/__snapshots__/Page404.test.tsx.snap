// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 Should match snapshot 1`] = `
<p>
  404 page not found! ;-( 
</p>
`;
